src/vocabularyChecker/detectWrongSyntax.test.ts

```diff
@@ -62,13 +62,13 @@ describe("logSingleWord", () => {
 		const position: Position = { fromIndex: 0, toIndex: 4 };
 		const issues = logSingleWord("Hello", position);
 		expect(issues).toEqual({
-			code: InvalidWord,
+			code: InvalidWord.INVALID_WORD.message,
 			message: `${"Hello"} von Stelle ${position.fromIndex} bis ${position.toIndex} ist ein unerlaubtes Wort! \n`,
```

Comment:
This is not the message defined in the JSON for this Issue.

Reply:
What is the right message then, if not InvalidWord.INVALID_WORD.message ?